docs/storm-sql-internal.md

```diff
@@ -1,59 +0,0 @@
----
```

Comment:
I removed the document since it is documented based on Trident implementation.

Reply:
@srdo 
Agreed. Regarding updating the doc, I guess most parts of content are correct, but unfortunately I don't have resource for diagram so required to redraw one. I'd rather explain in content to avoid redraw one (like "Now Trident topology in a diagram is replaced to normal Storm topology"). What do you think?

Reply:
It would probably be pretty quick to duplicate with e.g. https://www.draw.io/, since it's just some boxes.

Reply:
Yeah right. I didn't think about that. Drawing a new one.

Reply:
Just finished it.

Reply:
Thanks, looks great. I think the dotted line box is missing the "StormSQL" label. Also maybe we should put the draw.io source file in the docs directory too, so we don't have to remake the diagram again if we need to change it later.

Reply:
The second diagram on the page also mentions Trident. I don't know if it makes sense to update, or just remove it.

Reply:
Nice finding. Will fix it and also will attach source file of draw.io, but not sure where to add.
I'll draw for second diagram as well.